<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputWidget</class>
 <widget class="QWidget" name="InputWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QComboBox" name="obsTextSourceComboBox">
           <property name="toolTip">
            <string>&quot;Select a OBS text source to use its current text in the querystring or request body as `{{input}}` or `{{imageb64}}`.&quot;</string>
           </property>
           <item>
            <property name="text">
             <string>None</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_resizeInput">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Resize</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_resizeInput">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>100%</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>50%</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>25%</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>10%</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5%</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCheckBox" name="obsTextSourceEnabledCheckBox">
           <property name="text">
            <string>No empty</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="obsTextSourceSkipSameCheckBox">
           <property name="text">
            <string>No same</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>181</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="aggToTarget">
           <property name="toolTip">
            <string>Aggregate the input until empty is found, then use the aggregate value for the request.</string>
           </property>
           <property name="text">
            <string>Agg</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_aggTarget">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="currentText">
            <string>to Empty</string>
           </property>
           <item>
            <property name="text">
             <string>to Empty</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>for 30s</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>for 1m</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>for 2m</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>for 5m</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>for 10m</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
